{"uid":"ebbf1216daee92f","name":"test_video_handler[SciePro]","fullName":"lib.youtube.test_y2_video_handler#test_video_handler","historyId":"763bf1eaecc3b6b1140fcf33c684059a","time":{"start":1723826218874,"stop":1723826219702,"duration":828},"description":"\n    测试 youtu 视频剪辑\n    ","descriptionHtml":"<pre><code>测试 youtu 视频剪辑\n</code></pre>\n","status":"broken","statusMessage":"pytube.exceptions.PytubeError: Exception while accessing title of https://youtube.com/watch?v=B0lkyu2NFnA. Please file a bug report at https://github.com/pytube/pytube","statusTrace":"self = <pytube.__main__.YouTube object: videoId=B0lkyu2NFnA>\n\n    @property\n    def title(self) -> str:\n        \"\"\"Get the video title.\n    \n        :rtype: str\n        \"\"\"\n        if self._title:\n            return self._title\n    \n        try:\n>           self._title = self.vid_info['videoDetails']['title']\nE           KeyError: 'videoDetails'\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pytube/__main__.py:341: KeyError\n\nDuring handling of the above exception, another exception occurred:\n\ntest_output_dir = '/home/runner/work/github-runner/github-runner/mypy/tests/lib/youtube/_out/test_y2_video_handler'\nvideo_url = 'https://www.youtube.com/shorts/B0lkyu2NFnA'\n\n    @pytest.mark.parametrize(\n        'video_url',\n        ['https://www.youtube.com/shorts/B0lkyu2NFnA',\n         'https://www.youtube.com/shorts/3OSwmeCuq_0',\n         'https://www.youtube.com/watch?v=mAxFB3vbt7c',\n         'https://www.youtube.com/watch?v=c9hWEA9fiNQ',\n         'https://www.youtube.com/watch?v=MWIeswvua1I'],\n        ids=['SciePro', 'SlideSkills', 'TedEd', 'RegisteredNurseRN', 'vertical_video'])\n    def test_video_handler(test_output_dir, video_url):\n        \"\"\"\n        测试 youtu 视频剪辑\n        \"\"\"\n        work_path = os.path.join(test_output_dir, curr_func_name())\n>       api = Y2VideoHandler(work_path=work_path,\n                             channel_id=YouTube(video_url).channel_id,\n                             video_url=video_url,\n                             ffmpeg_log_output=True)\n\nmypy/tests/lib/youtube/test_y2_video_handler.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nmypy/lib/youtube/y2_video_handler.py:239: in __init__\n    self._video_original_title = y2_video.title\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pytube.__main__.YouTube object: videoId=B0lkyu2NFnA>\n\n    @property\n    def title(self) -> str:\n        \"\"\"Get the video title.\n    \n        :rtype: str\n        \"\"\"\n        if self._title:\n            return self._title\n    \n        try:\n            self._title = self.vid_info['videoDetails']['title']\n        except KeyError:\n            # Check_availability will raise the correct exception in most cases\n            #  if it doesn't, ask for a report.\n            self.check_availability()\n>           raise exceptions.PytubeError(\n                (\n                    f'Exception while accessing title of {self.watch_url}. '\n                    'Please file a bug report at https://github.com/pytube/pytube'\n                )\n            )\nE           pytube.exceptions.PytubeError: Exception while accessing title of https://youtube.com/watch?v=B0lkyu2NFnA. Please file a bug report at https://github.com/pytube/pytube\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pytube/__main__.py:346: PytubeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop_policy","time":{"start":1723815350385,"stop":1723815350385,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"copy_config_to_dir","time":{"start":1723826218871,"stop":1723826218873,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"test_output_dir","time":{"start":1723826218873,"stop":1723826218873,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"\n    测试 youtu 视频剪辑\n    ","status":"broken","statusMessage":"pytube.exceptions.PytubeError: Exception while accessing title of https://youtube.com/watch?v=B0lkyu2NFnA. Please file a bug report at https://github.com/pytube/pytube","statusTrace":"self = <pytube.__main__.YouTube object: videoId=B0lkyu2NFnA>\n\n    @property\n    def title(self) -> str:\n        \"\"\"Get the video title.\n    \n        :rtype: str\n        \"\"\"\n        if self._title:\n            return self._title\n    \n        try:\n>           self._title = self.vid_info['videoDetails']['title']\nE           KeyError: 'videoDetails'\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pytube/__main__.py:341: KeyError\n\nDuring handling of the above exception, another exception occurred:\n\ntest_output_dir = '/home/runner/work/github-runner/github-runner/mypy/tests/lib/youtube/_out/test_y2_video_handler'\nvideo_url = 'https://www.youtube.com/shorts/B0lkyu2NFnA'\n\n    @pytest.mark.parametrize(\n        'video_url',\n        ['https://www.youtube.com/shorts/B0lkyu2NFnA',\n         'https://www.youtube.com/shorts/3OSwmeCuq_0',\n         'https://www.youtube.com/watch?v=mAxFB3vbt7c',\n         'https://www.youtube.com/watch?v=c9hWEA9fiNQ',\n         'https://www.youtube.com/watch?v=MWIeswvua1I'],\n        ids=['SciePro', 'SlideSkills', 'TedEd', 'RegisteredNurseRN', 'vertical_video'])\n    def test_video_handler(test_output_dir, video_url):\n        \"\"\"\n        测试 youtu 视频剪辑\n        \"\"\"\n        work_path = os.path.join(test_output_dir, curr_func_name())\n>       api = Y2VideoHandler(work_path=work_path,\n                             channel_id=YouTube(video_url).channel_id,\n                             video_url=video_url,\n                             ffmpeg_log_output=True)\n\nmypy/tests/lib/youtube/test_y2_video_handler.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nmypy/lib/youtube/y2_video_handler.py:239: in __init__\n    self._video_original_title = y2_video.title\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pytube.__main__.YouTube object: videoId=B0lkyu2NFnA>\n\n    @property\n    def title(self) -> str:\n        \"\"\"Get the video title.\n    \n        :rtype: str\n        \"\"\"\n        if self._title:\n            return self._title\n    \n        try:\n            self._title = self.vid_info['videoDetails']['title']\n        except KeyError:\n            # Check_availability will raise the correct exception in most cases\n            #  if it doesn't, ask for a report.\n            self.check_availability()\n>           raise exceptions.PytubeError(\n                (\n                    f'Exception while accessing title of {self.watch_url}. '\n                    'Please file a bug report at https://github.com/pytube/pytube'\n                )\n            )\nE           pytube.exceptions.PytubeError: Exception while accessing title of https://youtube.com/watch?v=B0lkyu2NFnA. Please file a bug report at https://github.com/pytube/pytube\n\n/opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pytube/__main__.py:346: PytubeError","steps":[],"attachments":[{"uid":"82e50de0110f2362","name":"log","source":"82e50de0110f2362.txt","type":"text/plain","size":168}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"test_output_dir::0","time":{"start":1723826223817,"stop":1723826223817,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"copy_config_to_dir::finish_callback","time":{"start":1723826223817,"stop":1723826223818,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"youtube"},{"name":"parentSuite","value":"lib.youtube"},{"name":"suite","value":"test_y2_video_handler"},{"name":"host","value":"fv-az730-133"},{"name":"thread","value":"8891-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"lib.youtube.test_y2_video_handler"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"video_url","value":"'https://www.youtube.com/shorts/B0lkyu2NFnA'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["youtube"]},"source":"ebbf1216daee92f.json","parameterValues":["'https://www.youtube.com/shorts/B0lkyu2NFnA'"]}